pageTests/wards/book-a-visit.test.js

```diff
@@ -1,5 +1,10 @@
 import { getServerSideProps } from "../../pages/wards/book-a-visit";
 
+// TODO: This needs to be moved once the verifyToken logic is in the container..
```

Comment:
This can be removed since we're using the tokenProvider in the tests

Reply:
Done